tests/drivers/adc/adc_simple/src/main.c

```diff
@@ -36,7 +36,7 @@
 static u32_t seq_buffer[2][BUFFER_SIZE];
 
 static struct adc_seq_entry sample = {
-	.sampling_delay = 12,
+	.sampling_delay = 30,
```

Comment:
where does 30 come from?

Reply:
The sampling delay should be set to resolution of adc + 2.  In adc qmsi driver the sampling delay received form application is subtracted from  resolution of adc + 2 , if the resultant value is less than zero then zero is set as sampling delay. So in the adc_simple application when 12 is passed as sampling delay driver sets it as zero. So 26 is the minimum sampling delay should be set in application for a adc with resolution of 12. The reason to choose 30 was to maintain same value of sampling delay between adc_simple and adc_api application.

Reply:
ok, you should have this explanation in the commit message. That way we would know where this number is coming from.

Reply:
ok. I will add it

Reply:
Also please add some brief explanation what "sampling_delay " signify in this github discussion.

Reply:
ok. i will add it